### DISCUSSION TITLE
Support for setup on Heroku

### DISCUSSION THREAD

**Discussion by capriciousduck** [Kovah/LinkAce#374] [Support]
How to run this app on heroku?

**Comment by Kovah** [comment]
Hi. Please consult the Heroku documentation for help with running any Docker-based applications like LinkAce.

**Comment by capriciousduck** [comment]
Can you provide me the Dockerfile instead of the docker-compose? With the "Dockerfile" I can run it on Heroku. And how to make linkace work with $PORT env variable?

**Comment by Kovah** [comment]
The images can be found on the Docker Hub: https://hub.docker.com/r/linkace/linkace
LinkAce currently does not accept a $PORT environment variable.

**Comment by Kovah** [comment]
Were you able to use Linkace on Heroku?

**Comment by capriciousduck** [comment]
No.

**Comment by Kovah** [comment]
I will keep this open for later. Maybe I will find the time to setup a Heroku account for testing. Until then, any help from others is welcome.

**Comment by karimknaebel** [comment]
Heroku support would be nice indeed.

**Comment by debjit** [comment]
So I have made this tutorial on how you can run linkace on heroku dyno. Hope it helps.

https://debjit012.medium.com/deploy-laravel-w-laravel-mix-web-app-in-5-min-on-heroku-a1598cf70b6b

**Reply by debjit** [reply]
[Medium Tutorial](https://debjit012.medium.com/deploy-laravel-w-laravel-mix-web-app-in-5-min-on-heroku-a1598cf70b6b)
